frontend/src/AppRouter.tsx

```diff
@@ -133,11 +135,20 @@ const AppRouter: React.FC = () => {
         />
         <AppRoute
           protected
-          path="/lease/:leaseId?"
+          path="/lease/new"
+          exact
+          component={DetailContainer}
+          layout={AuthLayout}
+          claim={Claims.PROPERTY_VIEW}
```

Comment:
Is this the right claim? Would expect it to be `Claims.LEASE_CREATE`....

Reply:
True, will correct.